README.md

```diff
@@ -4,7 +4,15 @@ This is the parser for the [RecordFlux](https://github.com/Componolit/RecordFlux
 
 ## Development
 
-As a prerequisite to build the parser, [GNAT Community Edition](https://www.adacore.com/download) >= 2020 needs to be installed. To run the tests, the RecordFlux language package and its dependencies must also be installed. Either use the respective make target:
+As a prerequisite to build the parser, the following dependencies need to be installed:
+
+- [GNAT Community Edition](https://www.adacore.com/download) >= 2020
+- patchelf
+- libgmp-dev (Debian) or gmp-devel (Fedora)
```

Comment:
Do we really want to list concrete package names of different distributions? If so, Ubuntu (libgmp-dev) and Arch Linux (gmp) could be also mentioned.

Reply:
I think it can be helpful to list concrete packages. I added a generic explanation in addition to the concrete names for Debian/Ubuntu, Arch and Fedora.